### ISSUE TITLE
[feature request] multi camera view

### ISSUE THREAD

**Issue by mrglennjones** [b-jesch/plugin.video.ipcams#8]
is there a way to view multiple cameras at the same time?

**Comment by b-jesch** [comment]
No. That isn't possible because the video engine of kodi doesn't allow multiview (PIP or picture in picture).

**Comment by mrglennjones** [comment]
oh i see, thank you for the reply :)

**Comment by mrglennjones** [comment]
perhaps a compromise would be a cycle to next camera option in seconds?

**Comment by b-jesch** [comment]
Indeed. The addon "surveillance room" take advantage for this, but this addon isn't ported to Python 3 yet: https://github.com/maikito26/plugin.video.surveillanceroom

**Comment by mrglennjones** [comment]
thank you! i will check that out too
